packer/configs/gituserlist

```diff
@@ -0,0 +1,11 @@
+fauzigo
+philloooo
```

Comment:
I think we should get out those who are not working for CTDS anymore

Reply:
good catch :+1: